src/gsy_e/models/strategy/virtual_heatpump.py

```diff
@@ -72,24 +55,39 @@ def __init__(
         self.water_return_temp_C_profile_uuid = water_return_temp_C_profile_uuid
         self.dh_water_flow_m3_profile_uuid = dh_water_flow_m3_profile_uuid
 
-        VirtualHeatPumpValidator.validate(
-            maximum_power_rating_kW=maximum_power_rating_kW,
-            tank_parameters=tank_parameters,
-            water_supply_temp_C_profile=water_supply_temp_C_profile,
-            water_supply_temp_C_profile_uuid=water_supply_temp_C_profile_uuid,
-            water_return_temp_C_profile=water_return_temp_C_profile,
-            water_return_temp_C_profile_uuid=water_return_temp_C_profile_uuid,
-            dh_water_flow_m3_profile=dh_water_flow_m3_profile,
-            dh_water_flow_m3_profile_uuid=dh_water_flow_m3_profile_uuid,
-        )
+        # Repeat the validation for each of the tanks in order to conform to the validate method.
+        # TODO: Multiple tank validator will need to be created.
```

Comment:
IMO it is fine to loop here and keep as is.

Reply:
Thanks! Will remove the TODO in this case since you also agree